std/StandardToken.sol

```diff
@@ -21,11 +21,11 @@ contract StandardToken is Token {
 		return supply;
 	}
 
-	function transfer(address _to, uint256 _value) returns (bool success) {
```

Comment:
Hm, as you already also said at some point: It might be quite descriptive to keep these names (the variable might also just be named `failure`).
Perhaps we can count "return with value" as usage of the return variables?